packages/dlp/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "name": "dlp",
+  "version": "0.0.0",
+  "author": "Google Inc",
+  "description": "DLP API client for Node.js",
+  "main": "src/index.js",
+  "files": [
+    "src"
+  ],
+  "keywords": [
+    "google apis client",
+    "google api client",
+    "google apis",
+    "google api",
+    "google",
+    "google cloud platform",
+    "google cloud",
+    "cloud",
+    "google dlp",
+    "dlp",
+    "DLP API"
+  ],
+  "dependencies": {
+    "google-proto-files": "^0.12.0",
+    "google-gax": "^0.12.2",
```

Comment:
Will be updated in the generator with https://github.com/googleapis/googleapis/pull/350